ajax/libs/pagecss/package.json

```diff
@@ -0,0 +1,32 @@
+  {
+    "name": "pagecss",
+    "version": "0.3.0",
+    "filename": "page.min.css",
+    "author": "Andrew Chilton",
+    "keywords": [
+      "css",
+      "pagecss",
+      "front end"
+    ],
+    "description": "Beautiful CSS for single page sites - simple all the way up!",
+    "repository": {
+      "type": "git",
+      "url": "git://github.com/chilts/pagecss",
+      "dependencies": {
+        "node-sass": "^4.5.3"
+      }
+    },
+    "autoupdate": {
+      "source": "git",
+      "target": "git://github.com/chilts/pagecss.git",
+      "fileMap": [
+	      {
+         "basePath": "dist",
+  	  "files": [
+            "*/"
```

Comment:
On a side note, you can refer to https://github.com/isaacs/node-glob#glob-primer to learn more about how file rules work 👍